silkworm/infra/grpc/server/server.hpp

```diff
@@ -174,6 +176,9 @@ class Server {
     //! The gRPC server instance tied to this Server lifetime.
     std::unique_ptr<grpc::Server> server_;
 
+    //! The global callbacks are shared between all instances of Servers
+    inline static ServerGlobalCallbacks global_callbacks_{};
```

Comment:
```suggestion
    static inline ServerGlobalCallbacks global_callbacks_;
```

see https://github.com/erigontech/silkworm/blob/master/docs/code_style.md#p10-constants-definition-keywords